src/test/java/com/ably/kafka/connect/integration/ChannelSinkTaskTest.java

```diff
@@ -82,7 +98,7 @@ public void testMessagePublish_correctConfigAtLeastATaskIsRunning() throws Excep
     public void testMessagePublish_channelExistsWithStaticChannelName() {
         final String channelName = "test-channel";
 
-        connectCluster.kafka().createTopic(DEFAULT_TOPIC);
+
```

Comment:
Please remove this and all the other blank lines added in this PR :bow:

Reply:
https://github.com/ably/kafka-connect-ably/pull/61/commits/6ee1a15d5f2ce5ce8c190ec6e57c0e267e292d00

Reply:
ea8bcf079267c4f96b607f2b125fff92c7d15456